app/views/layouts/application.html.erb

```diff
@@ -13,5 +13,13 @@
 
   <body>
     <%= yield %>
+    <p class="navbar-text float-right">
+    <% if user_signed_in? %>
+      Logged in as <strong><%= current_user.email %></strong>.
+      <%= link_to 'My Listings', path="/apartmentindex", :class => 'navbar-link' %> |
+      <%= link_to "View Listings", path="/apartmentindex", :class => 'navbar-link'  %>
+
+    <% end %>
+    </p>
```

Comment:
The logic that is taking place here to have conditional rendering based on if a user is signed in is great, but should be located on your home page.  We should not have to modify any of the ruby embedded html views since we are utilizing React as our views inside the application.